hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2262,4 +2262,28 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    */
   boolean isSnapshotCleanupEnabled() throws IOException;
 
+
+  /**
+   * Retrieve online slowlog responses from in memory ringbuffer maintained by
```

Comment:
nit:
Retrieves
(since I see below you say "Clears" better to keep them consistent)